Source/FORMDataSource.m

```diff
@@ -939,9 +948,9 @@ - (void)insertDynamicSectionsForValues:(NSDictionary *)values
     [JSONAttributes enumerateKeysAndObjectsUsingBlock:^(NSString *key, id obj, BOOL *stop) {
         if ([obj isKindOfClass:[NSArray class]]) {
             FORMSection *section = [self sectionWithID:key];
-            [self.formsManager insertTemplateSectionWithID:key
-                                        intoCollectionView:self.collectionView
-                                                 usingForm:section.form];
+            for (NSInteger numberOfSections = 0; numberOfSections < [obj count]; numberOfSections++) {
+                [self.formsManager insertTemplateSectionWithID:key intoCollectionView:self.collectionView usingForm:section.form];
```

Comment:
Shouldn't this conform to our style guide?

https://github.com/hyperoslo/iOS-playbook/blob/master/STYLE_GUIDELINES.md